blocks/test/full-content.js

```diff
@@ -86,7 +86,7 @@ function normalizeParsedBlocks( blocks ) {
 	} );
 }
 
-describe( 'full post content fixture', () => {
+describe.skip( 'full post content fixture', () => {
```

Comment:
@gziolo This suite can probably be rewritten to use Jest snapshots.  What is the current issue with it?